src/pilot/config_idrac.py

```diff
@@ -129,6 +133,13 @@ def configure_nics_boot_settings(
                                                  pxe_nic_id)
 
 
+def skipped_configure_nics_boot_settings(drac_client, pxe_nic_id):
```

Comment:
Can you rename this method to something like "get_nic_mac_address"?

Reply:
done